BackEnd/src/main/java/com/unosquare/admin_core/back_end/controller/HolidayController.java

```diff
@@ -57,8 +67,69 @@ public HolidayViewModel findHolidayById(@PathVariable("eventId") int eventId) {
 
     @PostMapping(value = "/", produces = MediaType.APPLICATION_JSON_VALUE)
     @ResponseStatus(HttpStatus.CREATED)
-    public ResponseEntity<List<String>> createHoliday(@RequestBody CreateEventViewModel createEventViewModel) {
-        List<String> responses = createEventByType(createEventViewModel, EventTypes.ANNUAL_LEAVE);
+    public ResponseEntity createHoliday(@RequestBody CreateEventViewModel createHolidayViewModel) {
+
```

Comment:
The majority of this code lives in the base controller now. Anything new you've added needs to go in there instead. If not delete it out. Have you pulled the latest develop?